app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -274,7 +275,13 @@ class EditPatientScreen(context: Context, attributeSet: AttributeSet) : Relative
   }
 
   override fun showColonyOrVillagesList(colonyOrVillageList: List<String>) {
+    ArrayAdapter<String>(context, android.R.layout.simple_list_item_1, colonyOrVillageList).also { adapter ->
```

Comment:
Instead of using an Android layout file, let's create one in our project. We don't want to rely on that layout being the same across different OEMs.